beets/ui/commands.py

```diff
@@ -1613,6 +1614,10 @@ def modify_func(lib, opts, args):
     '-y', '--yes', action='store_true',
     help='skip confirmation'
 )
+modify_cmd.parser.add_option(
+    '-I', '--noinherit', action='store_false', dest='inherit', default=True,
+    help="Don't inherit album-changes to tracks"
```

Comment:
```suggestion
    help="when modifying albums, don't also change item data"
```

Just trying something even more explicit! To clarify that this does nothing when in item mode.